### PR TITLE
C++17 VPC support (+ on by default on Windows)

### PR DESCRIPTION
**Pull request by dimhotepus**
### Related Issue
N/A

### Implementation
* Fix VPC build on Windows. 
* Add C++14 / C++17 / C++ Latest support to VPC.
* Build/rebuild if needed VPC on projects creation first, than use build one for projects generation.
* Now no need to manually change language in generated for Visual Studio projects to C++17, it is default.
* Remove explicit setting of Visual Studio C++ project props which are already set to same by default - less large project files, easier to read them.

### Checklist

- [x] No other PRs implement this idea.
- [ ] This PR does not introduce any regressions.
- [x] I certify that this PR is my own entirely original work, or I have permission from and have attributed the relevant authors.
- [x] I have agreed to and signed this project's [Contributor License Agreement](https://cla-assistant.io/mastercomfig/team-comtress-2).
- [ ] This PR only contains changes to the engine and/or core game framework
- [x] This PR targets the `community` branch.

- [ ] This change has been filed as an issue.
- [x] No other PRs address this.
- [x] This PR is as minimal as possible.
- [ ] This PR does not introduce any regressions.
- [x] This PR has been built and locally tested on at least one platform.
- [ ] This PR has been tested on all platforms, on both a dedicated server and on a listen server.

### Testing Checklist

|         |            Client             |            Server             | Version                     |
|---------|:-----------------------------:|:-----------------------------:|-----------------------------|
| Windows | N/A | N/A |     |
|   Linux | N/A | N/A |  |
|  Mac OS | N/A | N/A |       |

### CODE REVIEW

**Comment by dimhotepus** [comment]
Tired of doing this by hands every time projects are regenerated.

Dear @mastercoms, could you please look at?

Simplifies working on https://github.com/mastercomfig/team-comtress-2/pull/554, also may be handy for master branch, too.

File: creategameprojects.bat [hunk 1]
```diff
@@ -1,2 +1,3 @@
 @echo OFF
+msbuild /m /p:Platform="x86" /p:Configuration=Release external/vpc/vpc.sln
```
Comment (Margen67) [inline creategameprojects.bat:2]:
```suggestion
msbuild /m /p:"Platform=x86;Configuration=Release" external/vpc/vpc.sln
```

File: creategameprojects_debug.bat [hunk 2]
```diff
@@ -1,2 +1,3 @@
 @echo OFF
+msbuild /m /p:Platform="x86" /p:Configuration=Release external/vpc/vpc.sln
```
Comment (Margen67) [inline creategameprojects_debug.bat:2]:
```suggestion
msbuild /m /p:Platform=x86;Configuration=Release external/vpc/vpc.sln
```

File: creategameprojects_dev.bat [hunk 3]
```diff
@@ -1,2 +1,3 @@
 @echo OFF
+msbuild /m /p:Platform="x86" /p:Configuration=Release external/vpc/vpc.sln
```
Comment (Margen67) [inline creategameprojects_dev.bat:2]:
```suggestion
msbuild /m /p:Platform=x86;Configuration=Release external/vpc/vpc.sln
```

File: creategameprojects.bat (hunk 1 above)
Comment (dimhotepus) [reply creategameprojects.bat]: Done, thanks!

File: creategameprojects_dev.bat (hunk 3 above)
Comment (dimhotepus) [reply creategameprojects_dev.bat]: Done, thanks!

File: creategameprojects_debug.bat (hunk 2 above)
Comment (dimhotepus) [reply creategameprojects_debug.bat]: Done, thanks!

**Comment by dimhotepus** [comment]
Closed, as too hard to review.

Slim PR is here: https://github.com/mastercomfig/team-comtress-2/pull/604

**Pull request closed without merging**